src/layouts/index.js

```diff
@@ -52,7 +50,7 @@ const DefaultLayout = props => {
   const { project } = useSiteMetadata();
 
   if (pageContext.layout === '404') {
-    return PageNotFoundLayout(props);
+    return PageNotFoundLayout(project, props);
```

Comment:
I supposed I missed this when you added this code, but one possible improvement could be to write this using JSX syntax, which is more in line with our approach in the rest of the repo:

```suggestion
    return <PageNotFoundLayout {...props} project={project} />
```